oio/cli/storage/obj.py

```diff
@@ -615,6 +615,13 @@ class LocateObject(ObjectCommandMixin, lister.Lister):
     def get_parser(self, prog_name):
         parser = super(LocateObject, self).get_parser(prog_name)
         self.patch_parser(parser)
+        parser.add_argument(
+            '--chunk-info',
+            action='store_true',
+            default=False,
+            help='Add the information of the chunk.'
+            'It creates a request per chunk located so it is likely to be slow'
```

Comment:
Display chunk size and hash as they are on persistent storage. It sends request per chunk so it is likely to be slow.